knowledge-base/how-tos/release.md

```diff
@@ -3,27 +3,32 @@ All SAP Cloud SDK modules will be published with the same version regardless whe
 
 ## Preparations for a Release
 
-Sometimes external libraries introduce breaking changing in minor version options.
-In order to ensure that the SDK works with the current versions of external libraries run `yarn install` to update the `yarn.lock`.
-If changes are there commit them to the main branch before releasing.
+Ensure that the changelog is up to date and correct.
+Move up high priority changes, so that they are easier to spot.
 
-## How to trigger a release
+## How to bump a version
 The release process can only be triggered by owners of the repository.
-If you are not in the owner list you will receive errors that the main branch is protected.
+If you are not in the owner list you will not be able to proceed.
 
-It is triggered by bumping the version using `lerna`. For convenience we added two scripts for patch and minor version bumps (`bump:patch` and `bump:minor`).
-Running one of those commands will increase the version and commit this change to the main branch. It will also create a version tag (e. g. `v1.18.0`), which in turn triggers a build to run.
-If this run is successful a Github release will be drafted. The name of the release will be the name of the tag.
+Releases are triggered by bumping the version using `lerna`.
+We have a github [workflow](https://github.com/sap/cloud-sdk-js/actions?query=workflow%3Abump) to do this.
+To trigger it, press "Run workflow" and enter "patch" or "minor" depending which version increase you want to have.
```

Comment:
Perhaps it's not possible to provide the link to the `Run workflow` button, because it's not merged. It would be great, if you can link later.

Reply:
I think it is generally not possible.